docs/developer-guide/building.md

```diff
@@ -183,60 +183,78 @@ components from `./bits` using the `./deps/eng/tools/bits-upload.sh` script.
      in your environment. If not set, `$UPDATES_IMGADM_CHANNEL` is computed
      automatically. See `./deps/eng/tools/bits-upload.sh`
 
-### Building from Gerrit
+### Useful development tools and building from GitHub Pull Requests
 
-When reviewing proposed changes from Gerrit, it can be useful to build and
-deploy those directly.
+Setting up the following tools is likely to make your work on Manta and Triton
+a little easier:
 
-Here we build patch set 3 of the `sdc-sapi.git` component, which has the
-gerrit id `5538`:
+* [hub](https://hub.github.com/) is a command line extension to git, with
+  particularly useful commands to list, checkout and create pull requests.
+* [jr](https://github.com/joyent/joyent-repos) is a tool that makes it
+  easier to interact with the many Joyent repositories you'll be working with
+  when developing Manta and Triton. It allows you to list and clone
+  repositories, as well as allowing you to run commands that affect multiple
+  repositories.
+* [prr](https://github.com/joyent/prr/) is a command line tool for merging an
+  approved pull request, allowing you to modify the commit message.
+
+When reviewing proposed changes from GitHub, it can be useful to build and
+deploy those directly allowing reviewers to exercise the changes before they're
+integrated.
+
+In this example, we'll use `jr` to clone the
+[sdc-manta](https://github.com/joyent/sdc-manta) repository, then we'll
```

Comment:
I'd drop the jr usage for cloning, frankly. While I can understand wanting to give a light intro to `jr` to encourage some more use of it, this isn't a compelling usage of `jr` -- it is just a bunch of wrapping around `git clone ...` that adds complexity for that task.

Reply:
Sure, I'll drop that example, but will still include jr in the list of tools.